libraries/aws_iam_managed_policies.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsIamManagedPolicies < AwsResourceBase
+  name 'aws_iam_managed_policies'
+  desc 'Verifies settings for a collection AWS Iam Policies'
```

Comment:
Change the description to "Lists all IAM users, groups, and roles that the specified managed policy is attached to."